src/resources/SearchUsageMetrics/SearchHubs/SearchHubsInterface.ts

```diff
@@ -47,6 +47,13 @@ export interface ListSearchHubsParams extends Paginated {
      * @default `0`
      */
     page?: number;
+
+    /**
+     * The minimum queries a search hub must have to be listed
+     * 
+     * @default '1'
```

Comment:
If its a number, it makes more sense to remove the singlequotes
```suggestion
     * @default 1
```